fastlane/report.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+  <testsuite name="fastlane.lanes">
+    
+    
+    
+      
+      <testcase classname="fastlane.lanes" name="0: default_platform" time="0.000389">
+        
+      </testcase>
+    
+      
+      <testcase classname="fastlane.lanes" name="1: clean assembleDebug assembleAndroidTest" time="81.298162">
+        
+      </testcase>
+    
+      
+      <testcase classname="fastlane.lanes" name="2: capture_android_screenshots" time="943.594079">
+        
+      </testcase>
+    
+  </testsuite>
+</testsuites>
```

Comment:
This strikes me as a generated file; do we really need it?

Reply:
it's already in gitignore

Reply:
git rm -rf fastlane/report.xml ?